README.rst

```diff
@@ -1,14 +1,82 @@
-A base package for multi-dimensional contiguious and non-contiguious spatially aware arrays.
---------------------------------------------------------------------------------------------
+`ndcube`
+========
 
-.. image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
-    :target: http://www.sunpy.org
-    :alt: Powered by Sunpy Badge
+|Latest Version| |Build Status| |Build status| |codecov| |Research
+software impact| |Powered by NumFOCUS| |Powered by Sunpy|
 
+.. |Powered by Sunpy| image:: http://img.shields.io/badge/powered%20by-SunPy-orange.svg?style=flat
+   :target: http://www.sunpy.org
+.. |Latest Version| image:: https://img.shields.io/pypi/v/ndcube.svg
+   :target: https://pypi.python.org/pypi/ndcube/
+.. |Build Status| image:: https://secure.travis-ci.org/sunpy/sunpy.svg
+   :target: http://travis-ci.org/sunpy/ndcube
+.. |Build status| image:: https://ci.appveyor.com/api/projects/status/github/sunpy/ndcube?branch=master&svg=true
+   :target: https://ci.appveyor.com/project/sunpy/ndcube
+.. |codecov| image:: https://codecov.io/gh/sunpy/ndcube/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/sunpy/ndcube
+.. |Powered by NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
+   :target: http://numfocus.org
 
+ndcube is a a open-source package for multi-dimensional contiguious and non-contiguious spatially aware arrays.
 
+Installation
+------------
+
+The stable version of ndcube can be installed simply via the anaconda distribution using the `conda-forge channel`:
+:: 
+    $ conda install --channel conda-forge ndcube
+or via pip:
+::
+    $ pip install ndcube
+
+If you want to to install the bleeding edge version or develop ndcube you will need to install from git.
+The best way to do this is to create a new conda environment and install the git
+version of ndcube in it:
+
+::
+
+    $ conda config --append channels conda-forge
+    $ conda create -n ndcube-dev python sunpy hypothesis pytest-mock
+    $ source activate ndcube-dev
+    $ git clone https://github.com/sunpy/ndcube.git ndcube-git
+    $ cd ndcube-git
+    $ pip install -e .
+
+For detailed (general) installation instructions, see the `installation guide`_ in
+the SunPy docs.
```

Comment:
Are the SunPy docs sufficient to give users more info here?  Or do we need an ndcube installation guide?

Reply:
It gives an overview of general installing. Conda setup and the like. It also walks through git install of sunpy. 

I would guess eventually that would be replaced with a link to the ndcube docs for installing it down the road.

Reply:
OK.  If we are happy to do that in the future and not now, then I approve these changes!

Reply:
Pending Travis and Appveyor passing, of course ;)

Thanks for this PR @nabobalis !

Reply:
No problem!